packer/gpu/buildkite-gpu-ami.pkr.hcl

```diff
@@ -0,0 +1,42 @@
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+
+source "amazon-ebs" "gpu_box" {
+  ami_name        = "buildkite-stack-linux-gpu-${local.timestamp}"
+  ami_description = "Buildkite Elastic Stack (Amazon Linux 2 LTS w/ nvidia-docker)"
+  ami_groups      = ["all"]
+  instance_type   = "g6.2xlarge"
+  launch_block_device_mappings {
+    delete_on_termination = true
+    device_name           = "/dev/xvda"
+    volume_size           = 50
+    volume_type           = "gp2"
+  }
+  region = var.region
+  source_ami_filter {
+    filters = {
+      architecture = "x86_64"
+      # Sync this with the default buildkite elastic stack AMI.
```

Comment:
provide a link or something on how to find the default buildkite ami?

Reply:
done